openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -38,6 +41,22 @@ def test_get_user_credentials(self):
         actual = get_user_credentials(self.user)
         self.assertEqual(actual, self.CREDENTIALS_API_RESPONSE['results'])
 
+    @httpretty.activate
+    def test_get_user_credentials_caching(self):
+        """Verify that when enabled, the cache is used."""
+        credential_config = self.create_credential_config(cache_ttl=1)
+        self.url = credential_config.internal_api_url + 'user_credentials/?username=' + self.user.username
```

Comment:
Call `strip()` on the internal API URL to avoid issues with trailing slashes, and include the slash in the string being appended.